packages/flipper-plugin-react-navigation/src/Logs.tsx

```diff
@@ -72,14 +73,12 @@ const Row = styled.button<{ selected: boolean; faded: boolean }>((props) => ({
   'fontSize': theme.monospace.fontSize,
   'textAlign': 'left',
   'padding': `${theme.space.medium}px ${theme.space.large}px`,
-  'color': props.selected ? '#fff' : '#000',
+  'color': props.selected ? '#fff' : '',
```

Comment:
Thanks for fixing this! It looks like this also includes the changes from #10001 which may cause approval issues for this PR. You probably want to make sure each PR is not dependent on the other.

Reply:
I took time to get back and fix the issue well. i closed one PR, and had both changes go in this one. As the changes are minimal. I appreciate checking the point in the description. And i set some demo video. ( i was using an RDP, so the rendering is a one of an RDP )

Reply:
I took time to get back and fix the issue well. i closed one PR, and had both changes go in this one. As the changes are minimal. I appreciate checking the point in the description. And i set some demo video. ( i was using an RDP, so the rendering is a one of an RDP )